pos_partner_deselection/static/description/index.html

```diff
@@ -0,0 +1,99 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan" style="color:#875A7B;">Partner deselection</h2>
+            <h3 class="oe_slogan">POS Partner deselection after a defined time interval</h3>
+        </div>
+    </div>
+</section>
+
+  <section class="oe_container oe_dark">
+  <div class="oe_row oe_spaced">
+    <h3 class="oe_slogan">How It Works</h3>
+    <div class="oe_span12 text-center">
+      <p class="oe_mt32">
+ 	<font style="font-size: 120%;">
+	  Administrator can edit POS settings and set up the Deselection interval (f.e. 10 sec)
+	</font>
+      </p>		
+    </div>
+    <div class="oe_screenshot" align="center">
+      <img style="max-width: 80%" src="deselection_interval.png"/>
+    </div>
+    <br/>
+    <div class="oe_span12 text-center">
+      <p class="oe_mt32">
+ 	<font style="font-size: 120%;">
+	  Then Administrator selects a Customer from the Catalog
+	</font>
+      </p>		
+    </div>
+    <div class="oe_screenshot" align="center">
+      <img style="max-width: 80%" src="set_customer.png"/>
+    </div>
+    <br/>
+    <div class="oe_span12 text-center">
+      <p class="oe_mt32">
+ 	<font style="font-size: 120%;">
+	  As a result, in opened POS the Customer is deselected after 10 seconds and Administrator can proceed with further work.
+	</font>
+      </p>		
+    </div>
+    <div class="oe_screenshot" align="center">
+      <img style="max-width: 80%" src="no_customer.png"/>
```

Comment:
I'd prefer to put the result at the beginning of the description and provide settings screenshot after that.

Reply:
Probably, the idea can be shown in a single screenshot. Say, "no_customer.png" or a screentshot with selected customer, but this a note that it will be delected in 10 sec (+ drawn icon of a timer ?)

Reply:
got it